SecureSDLCNotes.md

```diff
@@ -75,6 +75,27 @@ In order to better address security at the code level, it's recommended to follo
 ## Code Review
 Security code reviews are a manual or semi-manual process that leverages finding vulnerabilities in the code. It can be done by a security expert, or by a developer with security knowledge. To be more objective about the findings, it's recommended to use a checklist and/or a tool to automate the process. For example, when performing a PR review, the reviewer can use a checklist to ensure that the code is following one of the above secure coding standards.
 
+# Code Review References
+
+1. **Atlassian Article on Code Reviews:**
+   - Title: "Effective Code Review Guidelines"
+   - Source: Atlassian (Bitbucket) Blog
+   - [Effective Code Review Guidelines](https://www.atlassian.com/agile/software-development/code-reviews)
+
+ 
+
+2. **Google's Code Review Guide:**
+   - Title: "Google's Engineering Practices Documentation: Code Review"
+   - Source: Google Open Source
+   - [Google's Engineering Practices Documentation: Code Review](https://google.github.io/eng-practices/review/)
```

Comment:
Estas referencias no son específicas de revisiones de seguridad, son más prácticas generales.
Específico a las revisiones de código pero de seguridad sería algo como: https://www.synopsys.com/glossary/what-is-code-review.html